IFC4x3/Properties/b/BoltType_2VFVwrp3T3Sf2eWFjBM4gw/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="BoltType_2VFVwrp3T3Sf2eWFjBM4gw" Name="BoltType" UniqueId="9f3dfeb5-cc37-4372-90a8-80fb4b584aba" />
```

Comment:
Missing PrimaryDataType.

Reply:
Yes! This one should reuse the BoltType from #827. Just added it here to remember.